angular/src/app/landing/author/author-popup/author-popup.component.html

```diff
@@ -14,17 +14,17 @@ <h2 class="popup-header">Authors</h2>
         </div>
         <div class="ui-toolbar-group-right">
             <!-- Add author button -->
-            <button pButton type="submit" class="btn-add-author" style="background-color:rgb(221, 172, 9);"
+            <button pButton type="submit" id="btn-add-author" style="background-color:rgb(221, 172, 9);"
                 (click)="addAuthor();" label="Add Author" icon="faa faa-user-plus faa-1x icon-white" iconPos="left">
             </button>
 
             <!-- Cancel button -->
-            <button pButton type="submit" class="btn-cancel" (click)="activeModal.close('Close click')" label="Cancel"
+            <button pButton type="submit" id="btn-cancel" (click)="activeModal.close('Close click')" label="Cancel"
                 icon="faa faa-trash faa-1x icon-white" iconPos="left">
             </button>
 
             <!-- Save button -->
-            <button pButton type="submit" class="btn-Save" style="float:right; width:150px; background-color:green;"
```

Comment:
Is this class/id name case-sensitive?  I think it is; if so, it doesn't match the stylesheet.